serialization.go

```diff
@@ -138,13 +149,19 @@ func SendYAMLError(w http.ResponseWriter, _ *http.Request, err error) {
 
 // SendJSON sends a JSON response.
 // Declared as a variable to be able to override it for clients that need to customize serialization.
+// If serialization fails, it does NOT write to the response writer. It has to be passed to SendJSONError.
 var SendJSON = func(w http.ResponseWriter, _ *http.Request, ans any) error {
```

Comment:
do we want to to xml as well?

Reply:
Yes indeed, will be working on it tonight